### PR TITLE
Add roadmap

### PR DESCRIPTION
**Pull request by fgalan**
Fixes https://github.com/telefonicaid/iotagent-json/issues/379

@dcalvoalonso could you have a look and provide LGTM if ok, pls? Thanks!

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  out of the scope of GE owners. Thus, the moment in which these features will be
+  finalized cannot be assured.
+  
+### Short term
+
+The following list of features are planned to be addressed in the short term, and incorporated in a next release of the product:
+- Docker image protects username and password using environment variables.
+- Removal of warnings and deprecation warnings.
+- Creation of tutorial (to be contributed by members of the Community beyond GE owners)
+
```
Comment (dcalvoalonso) [inline docs/roadmap.md]: Fix of bugs reported in GitHub repository, merge of open PRs

File: docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Medium term
+
+The following list of features are planned to be addressed in the medium term, typically within the subsequent release(s) generated in the next 9 months after the next planned release:
```
Comment (dcalvoalonso) [inline docs/roadmap.md]: Deprecation of NGSIv1 could be also addressed in the medium term.

Comment (dcalvoalonso) [reply docs/roadmap.md]: Implementation of lazy attributes.

Comment (fgalan) [reply docs/roadmap.md]: Fixed in f1edcd59712b6069755a98d08bbfa277b69c5fac

File: docs/roadmap.md (hunk 1 above)
Comment (fgalan) [reply docs/roadmap.md]: Same as https://github.com/telefonicaid/iotagent-node-lib/pull/759#discussion_r258005583

NTC

**Pull request merged**